core/src/test/scala/cromwell/core/simpleton/WomValueBuilderSpec.scala

```diff
@@ -136,6 +136,57 @@ class WomValueBuilderSpec extends FlatSpec with Matchers with Mockito {
         WomValueSimpleton("object_with_object:b:bb[0]", WomString("bbrrrr")),
         WomValueSimpleton("object_with_object:b:bb[1]", WomString("bb gun")),
       )
+    ),
+    SimpletonConversion(
```

Comment:
ToL: Only because reading the WOM object notation is so awkward, could you add a quick comment explaining what this is simpletoning?